docs/esa/jwst/jwst.rst

```diff
@@ -225,68 +224,68 @@ To query the data products associated with a certain Observation ID
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> product_list=Jwst.get_product_list(observation_id='jw00777011001_02104_00001_nrcblong')
-  >>> for row in product_list:
-  >>>     print("filename: %s" % (row['filename']))
+  >>> product_list = Jwst.get_product_list(observation_id='jw01063107001_02101_00013_nrca3')
+  >>> print(product_list['filename'])
+
```

Comment:
doctest won't like empty lines like this and expects the output to be placed straight after the code lines.

Reply:
Fixed